docs/pages/index.mdx

```diff
@@ -5,55 +5,43 @@ description: Read an overview of the Teleport Access Platform. Learn how to impl
 tocDepth: 3
```

Comment:
GH isn't letting me add a comment to line 3, but we're still calling it the "Teleport Access Platform" in the main heading for this doc.
![image](https://github.com/user-attachments/assets/8c155e33-7dcd-4fff-86a7-f0b36baf2697)

Reply:
Confirmed with Eddie: this should be "Teleport Infrastructure Identity Platform"